services/core-api/app/api/mines/permits/permit_amendment/resources/permit_amendment_document.py

```diff
@@ -49,7 +49,7 @@ def put(self, mine_guid, permit_amendment_guid, permit_guid):
                 raise BadRequest('Must supply filename for new file upload.')
 
             new_pa_doc = PermitAmendmentDocument(
-                mine_guid=permit_amendment.permit.mine_guid,
+                mine_guid=permit_amendment.permit.mine.mine_guid,
```

Comment:
should this just be permit_amendment.mine_guid?

Reply:
changed.